app/views/articles/show.html.erb

```diff
@@ -128,10 +128,14 @@
             <% end %>
 
             <div class="crayons-article__subheader">
-              <a href="/<%= @user.username %>" class="flex items-center mr-4 mb-4 s:mb-0 fw-medium crayons-link">
-                <span class="crayons-avatar crayons-avatar--l mr-2"><img class="crayons-avatar__image" src="<%= Images::Profile.call(@user.profile_image_url, length: 50) %>" alt="<%= @user.username %> profile image" /></span>
-                <%= @user.name %>
+              <a href="/<%= @user.username %>" class="flex items-center mr-2 mb-4 s:mb-0 fw-medium crayons-link">
+                <span class="crayons-avatar crayons-avatar--l mr-2"><img class="crayons-avatar__image" src="<%= Images::Profile.call(@user.profile_image_url, length: 50) %>" alt="" /></span>
+                <span class="block m:hidden"><%= @user.name %></span>
```

Comment:
I'm assuming `display: none` here via `.hidden` is the way to go for a screen reader because we only want the screen reader to skip this if we're on desktop? And conversely for the Desktop view?

Reply:
That's the idea - we want the button to show on larger screens, the link to show in mobile. `display: none` will both visually remove and remove from the accessibility tree.